src/main/java/com/minecolonies/coremod/client/render/RenderArcherBarbarianMob.java

```diff
@@ -0,0 +1,50 @@
+package com.minecolonies.coremod.client.render;
+
+import com.minecolonies.coremod.client.model.ModelBarbarian;
+import com.minecolonies.coremod.entity.ai.mobs.EntityArcherBarbarian;
+import net.minecraft.client.model.ModelBiped;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraft.client.model.ModelBiped'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)